docs/develop/tools/contract-verifications.md

```diff
@@ -7,3 +7,18 @@ Contract verification refers to the process of auditing and verifying the code o
 
 It is typically performed by independent auditors, security experts, or automated tools. The process involves a thorough analysis of the contract code to identify potential issues, such as bugs, security vulnerabilities, or potential attacks. This can involve both manual and automated testing to ensure that the code meets certain standards and best practices.
 
+## [EScan](https://escan.live/)
+
+There are several block explorers serving the Evmos network and EScan is a service that allows block exploration as well as contract verification. For more information about the different block explorers, please proceed to our [block explorers's page](./explorers).
+
+[Link](https://escan.live/verifyContract) to verify and publish the contract source code.
+
+![escan-contract.png](/img/escan-contract.png)
+
+## [Mintscan](https://www.mintscan.io/evmos/evm)
+
+![mintscan-evm-contracts.png](/img/mintscan-evm-contracts.png)
```

Comment:
keep consistent order of images and descriptions for each tool, e.f. for Escan the image comes after the description.